__init__.py

```diff
@@ -327,6 +327,18 @@ def _init_setup_options(self):
                 {"name": "Dutch", "code": "nl", "system_code": "nl_NL"}
             ]
 
+        if "backend_list" not in self.settings:
+            # backend_name: display name to display in UI
+            # backend_icon: display icon to display in UI
+            # backend_type: backend type for configuration
+            self.settings["backend_list"] = [
+                {"backend_name": "No Backend", "backend_icon": "icons/nobackend.svg", "backend_type": "offline"},
+                {"backend_name": "Personal Backend", "backend_icon": "icons/personal.svg", "backend_type": "personal"},
+                {"backend_name": "OpenVoice Backend", "backend_icon": "icons/ovos.svg", "backend_type": "ovos"},
+                {"backend_name": "Neon Backend", "backend_icon": "icons/neongecko.svg", "backend_type": "neon"},
```

Comment:
can we leave Neon commented out for now? its not fully implemented yet, should only be added after https://github.com/OpenVoiceOS/ovos-backend-client/pull/11

Reply:
@NeonDaniel should this say `Neon Backend`,  `NeonGecko Backend` or  `NeonGeckoCom Backend` ?